src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -2675,6 +2677,11 @@ public void setLocale(@Nullable Locale locale) {
         settings.locale = locale == null ? null : locale.toLanguageTag();
     }
 
+    @Override
+    public @NonNull Identity identity() {
+        return Identity.identity(this.uuid);
```

Comment:
Should cache the `Identity` in a field, ideally, as this can end up being called often and is wasteful to create a new one each time.